src/layers/TrajservLayer.js

```diff
@@ -145,23 +141,23 @@ class TrajservLayer extends TrackerLayer {
    * @param {string} regexLine
    * @private
    */
-  static createFilter(line, trip, operator, regexLine) {
+  static createFilter(line, trip, operator, regexLine, isConstructor) {
```

Comment:
What's the point of isConstrcutor ?  It seems filterFc created in the constructor is overwritten in init() function by the `addTrackerFilters`.